requirements.lock

```diff
@@ -55,7 +55,7 @@ olefile==0.46
 packaging==19.0
 pathlib==1.0.1
 piffle==0.3.0
-Pillow==4.1.1
+# Pillow==4.1.1
```

Comment:
why is this commented out?

Reply:
This version of Pillow conflicted with `zlib`. I needed to install Pillow independently in order to get things operational